contrib/win32/install/server.wxs

```diff
@@ -0,0 +1,70 @@
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:firewall="http://schemas.microsoft.com/wix/FirewallExtension" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+    <Fragment>
+        <!-- KeyPath is necessary for multi-file components to identify the key file - preferrably versioned. -->
+        <ComponentGroup Id="Server" Directory="INSTALLFOLDER">
+            <ComponentGroupRef Id="Shared" />
+            <Component>
+                <File Name="sftp-server.exe" KeyPath="yes" />
+                <File Name="sftp-server.pdb" />
+            </Component>
+            <Component>
+                <File Name="ssh-shellhost.exe" KeyPath="yes" />
+                <File Name="ssh-shellhost.pdb" />
+            </Component>
+            <Component>
+                <File Id="sshd.exe" Name="sshd.exe" KeyPath="yes" />
+                <File Name="sshd.pdb" />
+                <RegistryKey Root="HKLM" Key="SOFTWARE\OpenSSH" ForceCreateOnInstall="yes">
```

Comment:
What happens if the registry HKLM:SOFTWARE\OpenSSH already exist but it has wrong permissions?
Does this logic, fix the registry permissions as defined here

Reply:
similar comment on line 20 
HKLM:SOFTWARE\OpenSSH\agent registry agent

Reply:
It should overwrite. If the installer - a TrustInstaller service - can't overwrite it, the machine is hosed and probably compromised anyway.